farmdata2_modules/fd2_tabs/fd2_barn_kit/transplantingReport.html

```diff
@@ -0,0 +1,639 @@
+                    border: 5px solid #d62a17;
+                    background-color: white;
+                    padding: 20px;
+                    margin: auto;
+                    width: 50%;
+                    text-align: center;
+                    font-size: large;">
+                    No Logs Found in These Dates 
+                </div> 
+            </div>
+        </div>
+    </div>
+    <script>
+        var seedingReport = new Vue ({
+            el: '#seedingReport',
+            components: {
+                'custom-table': CustomTableComponent,
+                'date-range-selection': DateRangeSelectionComponent,
+                'dropdown-with-all': DropdownWithAllComponent
+            },
+            data: {
+                startDate:'2020-05-01',//dayjs().startOf('year').format('YYYY-MM-DD').toString(),
+                endDate: dayjs().format('YYYY-MM-DD').toString(),
+
+                stillLoading: true,
+                reportVisible: false,
+                transplantingLogs: [],
+                updateLog: [],
+
+                sessionToken: null,
+                selectedCrop: 'All',
+                selectedArea: 'All',
+
+                headers: ['Date', 'Crop', 'Area', 'Bed Feet', 'Row Feet', 'Rows/Bed', 'Trays', 'Workers', 'Hours','Comments','User'],
+                visibleColumns: [true, true, true, true, true, true, true, true, true, true, true],
+
+                idToCropMap: new Map(),
+                cropToIDMap: new Map(),
+                idToUserMap: new Map(),
+                userToIDMap: new Map(),
+                areaToIDMap: new Map(),
+                unitToIDMap: new Map(),
+                configToIDMap: new Map(),
+
+                rowBeingEdited: false,
+                errBanner: null,
+            },
+            methods: {
+                hide(){
+                    this.reportVisible = false
+                    this.transplantingLogs = []
+                },
+                startDateChange(selectedDate){
+                    this.startDate = selectedDate
+                    //this.reportVisible = false
+                    //this.transplantingLogs = []
+                },
+                endDateChange(selectedDate){
+                    this.endDate = selectedDate
+                    //this.reportVisible = false
+                    //this.transplantingLogs = []
+                },
+                seedingLogRequestWithDates(){
+                    this.reportVisible = true
+                    let link = '/log.json?type=farm_transplanting&timestamp[ge]=' + dayjs(this.startDate).unix() + '&timestamp[le]=' + dayjs(this.endDate).unix()
+
+                    getAllPages(link, this.transplantingLogs)
+                        .then(() => {
+                            this.stillLoading = false
+                        })
+                        .catch((err) => { 
+                            this.errBanner = true
```

Comment:
Might be nice to factor this out into a helper method since it is used in a bunch of places.  That would eliminate code repetition and make it easier to read.

Reply:
This will not be included in this PR because we're considering creating a error component to handle all the error banners.